components/script/dom/htmlanchorelement.rs

```diff
@@ -604,7 +599,10 @@ pub fn follow_hyperlink(
     hyperlink_suffix: Option<String>,
     referrer_policy: Option<ReferrerPolicy>,
 ) {
-    // Step 1: TODO: If subject cannot navigate, then return.
+    // Step 1: If subject cannot navigate, then return.
```

Comment:
Without implementing the step, it's fine to leave the `TODO:` with the step description.
As we've implemented it now, I think it's good to just leave the `step` instead; we can check the latest step description from spec :)

```suggestion
    // Step 1
```